docs/proposals/pause-ensure-work/README.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Pause ensure work
+authors:
+- "@CharlesQQ"
+reviewers:
+- "@RainbowMango"
+approvers:
+
+creation-date: 2022-05-21
+
+---
+
+# Pause ensure work
+
+## Summary
+
+When the resource (as known as `resource template`) paused field(such as label `work.karmada.io/paused`)  is true, we hope that the latest change, whether it is workload or PP or OP, will not be updated to the work object until it becomes false
+
+## Motivation
+Users might add/replace/delete overriders for OP dynamically, and add/remove targetCluster in PP dynamically;
+These operations should be transactional, before all modifications are done, policy should not take effect  
```

Comment:
The `motivation` part focuses on the reason why raise the proposal and why people need it.